timelock-impl/src/main/java/com/palantir/atlasdb/timelock/adjudicate/PointHealthStatusReport.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.timelock.adjudicate;
+
+import java.util.Optional;
+
+import org.immutables.value.Value;
+
+@Value.Immutable
+public interface PointHealthStatusReport {
+    HealthStatus status();
+    Optional<String> message();
```

Comment:
Seems like we could just do the logging inside `getHealthStatusForMetric()`, unless you had something else in mind / are using these further down?

Reply:
Makes sense, will do